independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test/multipart/InvalidEncodingTest.java

```diff
@@ -57,7 +57,7 @@ public static class FeedbackResource {
         @Path("/multipart-encoding")
         @Produces(MediaType.TEXT_PLAIN)
         @Consumes(MediaType.MULTIPART_FORM_DATA + ";charset=UTF-8")
-        public String postForm(@MultipartForm final FeedbackBody feedback) {
+        public String postForm(@BeanParam final FeedbackBody feedback) {
```

Comment:
I don't really understand this change (and the similar changes to the other tests). Why `@BeanParam`?

Reply:
Because I deprecated `@MultipartForm` and wanted to test with and without the annotation.

Reply:
So `@BeanParam` is the new thing?

Reply:
It was already. Its meaning was always "group all `@Rest*` and `@*Param` into this parameter". It's just that RESTEasy classic decided to special-case multipart forms even though they used the regular `@FormParam` annotation and so they fit intuitively into the same logic as url-encoded forms, which already had existing semantics and behaviour as endpoint field, endpoint method parameter, and `@BeanParam`.